annyang.js

```diff
@@ -113,9 +113,9 @@
         if (result) {
           var parameters = result.slice(1);
           if (debugState) {
-            console.log('command matched: %c'+currentCommand.originalPhrase, debugStyle);
+            logMessage('command matched: %c'+currentCommand.originalPhrase);
             if (parameters.length) {
-              console.log('with parameters', parameters);
+              logMessage('with parameters "'+parameters.join('", "')+'"');
```

Comment:
Good point, of course, it was indeed one of two details that didn't feel entirely right to myself, at least not very elegant. The other was the second one you commented on.